app/src/main/java/org/openobservatory/ooniprobe/data/TestData.java

```diff
@@ -66,24 +63,50 @@ public static void doNetworkMeasurements(final Context ctx, final String testNam
         final String geoip_country = ctx.getFilesDir() + "/GeoIP.dat";
 
         final SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(ctx);
-
         final Boolean include_ip = preferences.getBoolean("include_ip", false);
         final Boolean include_asn = preferences.getBoolean("include_asn", true);
         final Boolean include_cc = preferences.getBoolean("include_cc", true);
         final Boolean upload_results = preferences.getBoolean("upload_results", true);
-        final String collector_address = preferences.getString("collector_address", OONITests.COLLECTOR_ADDRESS);
-        final String max_runtime = preferences.getString("max_runtime", OONITests.MAX_RUNTIME);
 
+        final String nameserver = DnsUtils.get_device_dns();
+        Log.v(TAG, "nameserver: " + nameserver);
```

Comment:
Isn't this variable unused?

Reply:
Probably has been there since we removed setting the nameserver to mk, right?